sherpa/ui/utils.py

```diff
@@ -10412,6 +10412,30 @@ def get_draws(self, id=None, otherids=(), niter=1000, covar_matrix=None):
             fit, covar_matrix, niter=niter)
         return (stats, accept, params)
 
+    def resample_data(self, id=None, niter=1000, seed=123):
+        """The function performs a parametric bootstrap assuming a skewed
+        normal distribution centered on the observed data point with the
+        variance given by the low and high measurement errors. The function
+        simulates niter realizations of the data and fits each realization
+        with the assumed model to obtain the best fit parameters. The function
+        returns the best fit parameters for each realization, and average and
+        standard deviation for the total number of realizations.
+
+        Example
+        -------
+        Account for of asymmetric errors when calculating parameter
+        uncertainties:
+
+        >>> load_ascii_with_errors(1,’test.dat’)
+        >>> set_model(‘polynom1d.p0’)
```

Comment:
The quotes here appear to be UTF-8 (or some other encoding) - which causes the Python 2.7 tests to fail. My guess is that it's the opening quote. Can we change to `'` for both (as it is likely to cause less problems with the documentation builds too).

Reply:
So, the error was (from the 0ea2317 run) https://travis-ci.org/sherpa/sherpa/jobs/547315894#L4113 which I was expecting to be fixed by @dtnguyen2 5a1ade4 commit. However, the 006d8a6 commit passes, which is before the fix. So I just don't understand things anymore.